pkg/rds/tags_test.go

```diff
@@ -0,0 +1,85 @@
+package rds
+
+import (
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/rds"
+	"github.com/aws/aws-sdk-go/service/rds/rdsiface"
+)
+
+type mockRdsService struct {
+	rdsiface.RDSAPI
+}
+
+func (m *mockRdsService) DescribeDBClusters(input *rds.DescribeDBClustersInput) (*rds.DescribeDBClustersOutput, error) {
+	dbName := *input.DBClusterIdentifier
+
+	if dbName == "unknown" || dbName == "instance" {
+		return &rds.DescribeDBClustersOutput{}, nil
+	}
+
+	dbClusters := make([]*rds.DBCluster, 1)
+	dbClusters[0] = &rds.DBCluster{
+		DBClusterArn: aws.String("arn:aws:rds:us-east-1:123456789012:db:" + dbName),
+	}
+	return &rds.DescribeDBClustersOutput{
+		DBClusters: dbClusters,
+	}, nil
```

Comment:
something like this might be better here:

```golang
	return &rds.DescribeDBClustersOutput{
		DBClusters: []*rds.DBCluster{
			&rds.DBCluster{
				DBClusterArn: aws.String("arn:aws:rds:us-east-1:123456789012:db:" + dbName)
			}
		}
	},nil
```